src/components/v5/common/Filter/partials/DesktopFilter.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FC } from 'react';
+import { usePopperTooltip } from 'react-popper-tooltip';
+
+import { DesktopFilterProps } from '../types';
+import FilterButton from '~v5/shared/Filter/FilterButton';
+import FilterOptions from './FilterOptions';
+import { filterOptions } from '../consts';
+
+const displayName = 'v5.common.Filter.partials.DesktopFilter';
+
+const DesktopFilter: FC<DesktopFilterProps> = ({ isOpened, setOpened }) => {
```

Comment:
I am not sure if the DesktopFilter name of the component is correct. Couldn't be 'FilterPopper' or 'FilterTooltip'or something like that?